testing/internal/inventorysorter/pkgmatcher.go

```diff
@@ -0,0 +1,20 @@
+// Package inventorysorter provides a Sort function for inventories to allow
+// clear comparisons with cmp.Diff()
+package inventorysorter
```

Comment:
Let's rename the file to inventorysorter.go too

Reply:
Removed completely following other comment.